specs/core/0_beacon-chain.md

```diff
@@ -502,10 +495,12 @@ The following data structures are defined as [SimpleSerialize (SSZ)](https://git
 {
     # Slot number
     'slot': 'uint64',
-    # Shard number (`BEACON_CHAIN_SHARD_NUMBER` for beacon chain)
-    'shard': 'uint64',
```

Comment:
Question 🙋‍♀️: does that mean we can remove `shard` from `BeaconBlock` either with or without this PR?

Reply:
Yes. Although it's natural to separate the beacon and shard proposer slashing conditions with this PR because we now have a notion of header. See discussion [here](https://github.com/ethereum/eth2.0-specs/pull/633#discussion_r257070067).